Tests/MeiliSearchIntegrationTests/KeysTests.swift

```diff
@@ -16,28 +16,25 @@ class KeysTests: XCTestCase {
     super.setUp()
 
     session = URLSession(configuration: .ephemeral)
-    client = try! MeiliSearch(host: "http://localhost:7700", apiKey: "masterKey", session: session)
+    client = try! MeiliSearch(host: currentHost(), apiKey: "masterKey", session: session)
+
+    let semaphore = XCTestExpectation(description: "Setup: delete all keys")
```

Comment:
I thought `XCTestExpectation` should be used for expectations only inside of test cases, but it seems to be the recommended way from Apple to do async operations in the setup/teardown hooks.

The weirdest issue is related to the multi-env problem. When I ran that old configuration on Mac no issue at all. But when I ran locally on Docker, I got flaky tests! 😮‍💨

Reply:
There are still flacky tests in docker ?

Reply:
No, as far as I know!